NEWS.md

```diff
@@ -1,3 +1,11 @@
+# dv.loader 2.1.0
+
+- Refactored code to improve readability and maintainability.
+
+- Fixed issue of partial matching when the `file_names` argument contains no file extensions.
```

Comment:
Is this the case? Looking at the code, I can't tell. What is the old behavior and what is the new one?

Reply:
The new function load_data_files() requires full file paths to ensure exact matching, avoiding the potential file loading errors that could occur with load_data() when file extensions are omitted.

Reply:
You mention you've fixed the partial matching of `file_names`. That argument appears in the legacy `load_data` and the new `get_file_paths`. If the solution to the partial matching is to use `load_data_files`, then mentioning the `file_names` argument is misleading, because that function doesn't have an argument by that name.

Reply:
The partial matching issue has been addressed in two ways. First, the new `load_data_files()` function requires full file paths with extensions, eliminating any ambiguity. Second, for users of the legacy `load_data()` function, the new `print_file_paths = TRUE` option helps identify potential file name matching issues by displaying the complete paths of files being loaded.